manifests/c/CloudImperiumGames/RSILauncher/1.4.11/CloudImperiumGames.RSILauncher.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+PackageIdentifier: "CloudImperiumGames.RSILauncher"
+PackageVersion: "1.4.11"
+PackageLocale: "en-US"
+Publisher: "Cloud Imperium Games"
+PublisherUrl: "https://www.cloudimperiumgames.com/"
+PublisherSupportUrl: "https://support.robertsspaceindustries.com/hc/en-us"
+PrivacyUrl: "https://robertsspaceindustries.com/privacy"
+PackageName: "RSI Launcher"
+PackageUrl: "https://www.robertsspaceindustries.com/"
+License: "Copyright (c) Cloud Imperium Rights LLC and Cloud Imperium Rights Ltd"
+LicenseUrl: "https://robertsspaceindustries.com/eula"
+Copyright: "Copyright (c) Cloud Imperium Rights LLC and Cloud Imperium Rights Ltd"
+CopyrightUrl: "https://robertsspaceindustries.com/eula"
+ShortDescription: "The Star Citizen Installer is your entry point into all the released modules, which we'll be constantly expanding as we move toward the completed game."
+Moniker: "rsilauncher"
+Tags: 
+- "Star-Citizen"
+- "RSI-Launcher"
+- "Games"
+- "Gaming"

```

```suggestion
- "star-citizen"
- "rsi-launcher"
- "games"
- "gaming"
```
As per the latest documentation, all of these tags needs to be in lowercase.